src/main/java/pl/betoncraft/betonquest/events/PointEvent.java

```diff
@@ -88,12 +95,24 @@ private void addPoints(final String playerID, final PlayerData playerData) throw
         if (multi) {
             for (final Point p : playerData.getPoints()) {
                 if (p.getCategory().equalsIgnoreCase(category)) {
-                    playerData.modifyPoints(category,
-                            (int) Math.floor((p.getCount() * count.getDouble(playerID)) - p.getCount()));
+                    playerData.modifyPoints(category, (int)Math.floor(p.getCount() * count.getDouble(playerID) - p.getCount()));
+                    if (notify) {
+                        Config.sendNotify(playerID, "point_multiply", new String[] { String.valueOf(count.getInt(playerID)), categoryName }, "point_multiply,info");
+                    }
+                }
+            }
+        }
+        else {
+            playerData.modifyPoints(category, (int)Math.floor(count.getDouble(playerID)));
+            if (notify) {
+                if (count.getInt(playerID) > 0) {
+                    Config.sendNotify(playerID, "point_given", new String[] { String.valueOf(count.getInt(playerID)), categoryName }, "point_given,info");
```

Comment:
Maybe extract `count.getInt(playerID)` to its own local variable?

Reply:
I would recommend to combine the bool expression here to reduce nesting.
```java
if (notify && count.getInt(playerId) > 0) { ... }
else { ... }
```

Reply:
```java
if (condition1) {
    if (condition2) {
        // codeA
    } else {
        // codeB
    }
}
```
isn't equal to
```java
if (condition1 && condition2) {
    // codeA
} else {
    // codeB
}
```

Reply:
yes, i know. So I shouldn't change it or do it like this
`if (condition1 && condition2) {
    // codeA
} else if (condition1) {
    // codeB
}`

Reply:
Dont change it. Just change the condition from a direct comparision of two integers to a boolean. 
 See my other comment for more information.

Reply:
I'm sorry, this is correct. Wrote my comment too late yesterday 🤦‍♂️